turbinia/pubsub.py

```diff
@@ -60,20 +60,22 @@ def setup(self):
   def setup_publisher(self):
     """Set up the pubsub publisher."""
     config.LoadConfig()
-    self.publisher = pubsub.PublisherClient()
+    self.publisher = gcp_common.CreateService('pubsub', 'v1')
```

Comment:
maybe call this self.pubsub_client as it will be used for any pubsub action and not only publishing
```suggestion
    self.pubsub_client = gcp_common.CreateService('pubsub', 'v1')
```